templates/server-statefulset.yaml

```diff
@@ -83,11 +83,11 @@ spec:
           {{ template "vault.mounts" . }}
           ports:
             - containerPort: 8200
-              name: http
+              name: {{ include "vault.scheme" . }}
             - containerPort: 8201
-              name: internal
+              name: {{ include "vault.scheme" . }}-internal
             - containerPort: 8202
-              name: replication
+              name: {{ include "vault.scheme" . }}-rep
```

Comment:
Same question for the internal the replication ports here.

Reply:
same answer, pretty much just consistency. the only problem i had that was solved was changing 8200 to http-8200. the rest is consistency.